### PR TITLE
Selenium 3

### PR DESCRIPTION
**Pull request by Rushabhshroff**

### CODE REVIEW

File: config/local.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "capabilities": {
+      "project": "Pytest Browserstack",
+      "build": "pytest-browserstack",
+      "name": "local_test",
```
Comment (Ankit098) [inline config/local.json:5]: name as `BStack pytest local` (we want to keep BStack in the test name for sample repos..)

File: config/local.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  },
+  "environments": [
+      {
+          "os": "Windows",
+          "os_version": "11",
+          "browser": "Chrome",
+          "browser_version": "latest"
+      }
+  ]
+}
```
Comment (Ankit098) [inline config/local.json:18]: EoF fixes on a few files..

File: config/local.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "capabilities": {
+      "project": "Pytest Browserstack",
+      "build": "pytest-browserstack",
```
[the remark this answers is not in the document]
Comment (Rushabhshroff) [reply config/local.json]: Done https://github.com/browserstack/pytest-browserstack/pull/3/commits/70f4be6802d0d330065b74df033219fdc3a9921b

File: config/parallel.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+    "capabilities": {
+        "project": "Pytest Browserstack",
+        "build": "pytest-browserstack",
+        "name": "local_test",
```
[the remark this answers is not in the document]
Comment (Rushabhshroff) [reply config/parallel.json:5]: Done https://github.com/browserstack/pytest-browserstack/pull/3/commits/70f4be6802d0d330065b74df033219fdc3a9921b

File: config/parallel.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
+{
+    "capabilities": {
+        "project": "Pytest Browserstack",
+        "build": "pytest-browserstack",
```
[the remark this answers is not in the document]
Comment (Rushabhshroff) [reply config/parallel.json]: Done https://github.com/browserstack/pytest-browserstack/pull/3/commits/70f4be6802d0d330065b74df033219fdc3a9921b

File: requirements.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+browserstack-local
+jsonmerge
+multiprocess
+paver
+selenium
```
[the remark this answers is not in the document]
Comment (Rushabhshroff) [reply requirements.txt]: Done https://github.com/browserstack/pytest-browserstack/pull/3/commits/70f4be6802d0d330065b74df033219fdc3a9921b

File: config/local.json (hunk 1 above)
Comment (Rushabhshroff) [reply config/local.json]: Done https://github.com/browserstack/pytest-browserstack/pull/3/commits/f6f548fa2e0fba0f738e7e794ee9bced345e05f8

File: config/local.json (hunk 2 above)
Comment (Rushabhshroff) [reply config/local.json]: Done https://github.com/browserstack/pytest-browserstack/pull/3/commits/70f4be6802d0d330065b74df033219fdc3a9921b

File: config/single.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
+{
+    "capabilities": {
+        "project": "Pytest Browserstack",
+        "build": "browserstack-build-1",
+        "name": "single_test",
```
Comment (Ankit098) [inline config/single.json:5]:
```suggestion
        "name": "BStack pytest single",
```

File: config/parallel.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
+{
+    "capabilities": {
+        "project": "Pytest Browserstack",
+        "build": "browserstack-build-1",
+        "name": "local_test",
```
Comment (Ankit098) [inline config/parallel.json:5]:
```suggestion
        "name": "BStack pytest parallel",
```

**Review by Ankit098 — APPROVED** [review]

**Pull request merged**